game/src/core/components/PlayerComponent.java

```diff
@@ -60,21 +62,22 @@ public Optional<Consumer<Entity>> registerFunction(int key, Consumer<Entity> fun
     /**
      * Remove the registered function on the given key.
      *
-     * @param key Value of the key.
+     * @param key The integer value of the key.
+     * @see com.badlogic.gdx.Gdx#input
      */
-    public void removeFunction(int key) {
+    public void removeFunction(final int key) {
         functions.remove(key);
     }
 
     /**
-     * Will check each entry in the function map, and if the key is just pressed, the function will
-     * be executed.
+     * This method will check each entry in the function map, and if the key is just pressed, the
```

Comment:
https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment

"It does not begin with [...] This method returns... [...]"

Reply:
Vorschlag: "Execute the callback function registered to a key when it is pressed."